### DISCUSSION TITLE
Calculate incremental strain

### DISCUSSION THREAD

**Discussion by 123fgh123** [damask-multiphysics/DAMASK#43] [Q&A] [answered]
Is it possible to calculate the incremental equivalent plastic strain using the built-in methods “add_calculation(formula, name, unit='n/a', description=None)” ？ https://www.damask.mpie.de/_modules/damask/_result.html#Result.add_calculation

**Comment by MarDiehl** [comment, marked as answer]
`add_calculation`, like all the other `add_xxx` methods, works only with datasets within the same group (the equivalent of a folder in HDF5). If you need to calculate the increase in strain over time, you need to obtain the strain per increment and subtract from each other in Python

**Comment by 123fgh123** [comment]
Ok, Thank you.
